aptos-move/aptos-vm/src/move_vm_ext/warm_vm_cache.rs

```diff
@@ -101,6 +101,7 @@ struct WarmVmId {
     natives: Bytes,
     vm_config: Bytes,
     core_packages_registry: Option<Bytes>,
+    v2_flag: bool,
```

Comment:
Is this for loading up compiled results for compiler v2? This field itself doesn't quite make sense to me and shouldn't go into production?

Reply:
It is used to differentiate between V1 and V2 bytecode in the warmvm cache. Do you have any suggestions on how to make it less hacky?

Reply:
Updated the code to make it less hacky